languages/common-lisp/exercises/concept/lists/.docs/introduction.md

```diff
@@ -1,18 +1,12 @@
-### Lists in Common Lisp
+### lists
```

Comment:
We might want title case (like Lists)? I'm not sure if this is supposed to exactly match the slug though... I could just be a bit behind on the guidelines.

Reply:
Looks like just a typo - thanks for catching.